apps/full-site-editing/full-site-editing-plugin/full-site-editing-plugin.php

```diff
@@ -52,6 +52,75 @@ function load_full_site_editing() {
 }
 add_action( 'plugins_loaded', __NAMESPACE__ . '\load_full_site_editing' );
 
+/**
+ * Whether or not FSE is active.
+ * If false, FSE functionality can be disabled.
+ *
+ * @returns bool True if FSE is active, false otherwise.
+ */
+function is_full_site_editing_active() {
+	return is_site_eligible_for_full_site_editing() &&
+			( is_theme_supported() || in_array( get_theme_slug(), SUPPORTED_THEMES, true ) );
+}
+
+/**
+ * Returns normalized theme slug for the current theme.
+ *
+ * Normalize WP.com theme slugs that differ from those that we'll get on self hosted sites.
+ * For example, we will get 'modern-business-wpcom' when retrieving theme slug on self hosted sites,
+ * but due to WP.com setup, on Simple sites we'll get 'pub/modern-business' for the theme.
+ *
+ * @return string Normalized theme slug.
+ */
+function get_theme_slug() {
```

Comment:
This can probably go as this can be handled by a filter in the given context with this approach